cgmanifest.json

```diff
@@ -0,0 +1,167 @@
+{
+  "Registrations": [
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "0831265c1aca95ca02eca5bf1155e4251e545328",
+          "repositoryUrl": "https://github.com/microsoft/BlingFire.git"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "v19.22",
+          "repositoryUrl": "https://github.com/davisking/dlib.git"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "2020-11-01",
+          "repositoryUrl": "https://github.com/google/re2.git"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "release-1.11.0",
+          "repositoryUrl": "https://github.com/google/googletest.git"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "v3.7.3",
+          "repositoryUrl": "https://github.com/nlohmann/json.git"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "4.5.4",
```

Comment:
opencv has a lot of dependencies. For example, libpng. If it was used when you build your package, then you need to list it here as well.  Transitively. 
Without the dependencies, opencv itself doesn't do any useful thing.

Reply:
Okay! I will cover the transitive dependencies as well. How accurate should be versions be? When the requirement is a range which should be included, the lower or upper?

Reply:
Should be exactly the version that you use for building your packages. Exactly. Not a range.